BLSQ/iaso
# POLIO-1003: Frontend: remove surge data from preparedness tab

Explain what problem this PR is resolving
- Surge datas(fields) are not needed anymore
Related JIRA tickets : [POLIO-1003](https://bluesquare.atlassian.net/browse/POLIO-1003)

## Self proofreading checklist


## Changes
- Remove all field related to surge on frontend

## How to test
- Go to campaigns
- Edit or create a campaign
- Go to Preparedness tab
- Check if the surge field still there

## Print screen / video
### Before the changes
![Screenshot from 2023-04-17 19-04-30](https://user-images.githubusercontent.com/19631540/236179111-a419d4f7-484b-4712-8a56-15ca2f8d23fa.png)
### After the changes
![Screenshot from 2023-04-17 19-05-33](https://user-images.githubusercontent.com/19631540/236179200-38ec39ed-0b70-4974-a0f8-d439c54bcfa4.png)



[POLIO-1003]: https://bluesquare.atlassian.net/browse/POLIO-1003?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
